README.rst

```diff
@@ -75,13 +80,42 @@ Once tags are applied to a build successfully, a message will be sent to topic
         "context": context,
       },
       "nvr": N-V-R,
-      "destination_tags": [tag1, tag2, ...]
+      "destination_tags": [
+        {'tag': name_1, 'task_id': 1},
```

Comment:
Let's use double quotes here so it's valid Json.

Reply:
Done.